examples/datasource_types/prometheus.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: grafana.integreatly.org/v1beta1

```

Comment:
Do we need the prometheus option? I think similar is avliable in other examples.

Reply:
Hmm, maybe it's still worth keeping it here as a separate datasource example, don't think this is a blocker for merging though, right?